packages/lesswrong/server/repos/PostViewTimesRepo.ts

```diff
@@ -49,20 +49,57 @@ class PostViewTimesRepo extends IncrementalViewRepo<"PostViewTimes"> {
         post_id,
         date_trunc('day', timestamp)
     `,
-      [windowStart, windowEnd]
+      [startDateFormatted, endDateFormatted]
     );
 
-    // Manually convert the number fields from string to number (as this isn't handled automatically)
+    // Manually convert:
+    // - The number fields from string to number (as this isn't handled automatically)
+    // - The date fields to UTC strings (NOTE: this actually changes the time represented, because in the conversion
+    //   from postgres to JS the timezone of the server is applied to the raw date, so 2023-04-01 00:00:00 becomes e.g.
+    //   2023-04-01 00:00:00+0100 (BST), which is wrong)
```

Comment:
This issue caused the data to be offset by an hour on insert, but not actually be wrong (and because of the way we select from the tables, this didn't  have any effect)

Reply:
No, because `windowStart` going in would be something like `2023-04-01 00:00:00+0100` (i.e. midnight, but in the server timezone), so converting to utc would keep the actual global time the same and result in `2023-03-31 23:00:00`